src/lib/yang/parser.lua

```diff
@@ -283,8 +283,10 @@ function Parser:parse_statement()
    -- Take the identifier
    self:consume_whitespace()
    local argument = self:parse_string()
-   if argument ~= "" then returnval.argument = argument end
    self:skip_whitespace()
+   if self:peek() == ";" or self:peek() == "{" then
```

Comment:
I think the tests will fail with this code.  Consider replacing lines 283-289 with:

```lua
if self:skip_whitespace() and self:peek() ~= ';' and self:peek() ~= '{' then
   returnval.argument = self:parse_string()
   self:skip_whitespace()
end
```